api/openapi.yaml

```diff
@@ -1945,6 +2282,31 @@ components:
             type: string
           nullable: true
 
+    AssetScanEstimationState:
+      type: object
+      properties:
+        state:
```

Comment:
Maybe worth adding a description, saying that in order to initiate an estimation, need to create it in pending state

Reply:
I think its worth handling this in the AssetScanEstimation watcher. If it sees an AssetScanEstimation with no state, set it to Pending, PATCH it in the DB and then continue to process it as though it was in Pending.

Reply:
Should we do this for scanEstimation as well?

Reply:
I think so, eventually we should do it for AssetScan/Scan objects too I think.